trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/deflect_control_surface.py

```diff
@@ -0,0 +1,619 @@
+
+
+    Properties Used:
+    N/A
+    """    
+    
+    # Unpack
+    vertical_wing = wing.vertical
+    inverted_wing = wing.inverted_wing
+    
+    x_origin, y_origin, z_origin = wing.origin[0]
+    
+    y_origin *= sym_sign
+    
+    xi_prime_a1    = raw_VD.xi_prime_a1   
+    xi_prime_ac    = raw_VD.xi_prime_ac   
+    xi_prime_ah    = raw_VD.xi_prime_ah   
+    xi_prime_a2    = raw_VD.xi_prime_a2   
+    y_prime_a1     = raw_VD.y_prime_a1    
+    y_prime_ah     = raw_VD.y_prime_ah    
+    y_prime_ac     = raw_VD.y_prime_ac    
+    y_prime_a2     = raw_VD.y_prime_a2    
+    zeta_prime_a1  = raw_VD.zeta_prime_a1 
+    zeta_prime_ah  = raw_VD.zeta_prime_ah 
+    zeta_prime_ac  = raw_VD.zeta_prime_ac 
+    zeta_prime_a2  = raw_VD.zeta_prime_a2 
+                                          
+    xi_prime_b1    = raw_VD.xi_prime_b1   
+    xi_prime_bh    = raw_VD.xi_prime_bh   
+    xi_prime_bc    = raw_VD.xi_prime_bc   
+    xi_prime_b2    = raw_VD.xi_prime_b2   
+    y_prime_b1     = raw_VD.y_prime_b1    
+    y_prime_bh     = raw_VD.y_prime_bh    
+    y_prime_bc     = raw_VD.y_prime_bc    
+    y_prime_b2     = raw_VD.y_prime_b2    
+    zeta_prime_b1  = raw_VD.zeta_prime_b1 
+    zeta_prime_bh  = raw_VD.zeta_prime_bh 
+    zeta_prime_bc  = raw_VD.zeta_prime_bc 
+    zeta_prime_b2  = raw_VD.zeta_prime_b2 
+                                          
+    xi_prime_ch    = raw_VD.xi_prime_ch   
+    xi_prime       = raw_VD.xi_prime      
+    y_prime_ch     = raw_VD.y_prime_ch    
+    y_prime        = raw_VD.y_prime       
+    zeta_prime_ch  = raw_VD.zeta_prime_ch 
+    zeta_prime     = raw_VD.zeta_prime    
+
+    # flip over y = z for a vertical wing since deflection math assumes horizontal wing--------------------
+    y_prime_a1, zeta_prime_a1 = flip_1(y_prime_a1, zeta_prime_a1, vertical_wing, inverted_wing)
+    y_prime_ah, zeta_prime_ah = flip_1(y_prime_ah, zeta_prime_ah, vertical_wing, inverted_wing)
+    y_prime_ac, zeta_prime_ac = flip_1(y_prime_ac, zeta_prime_ac, vertical_wing, inverted_wing)
+    y_prime_a2, zeta_prime_a2 = flip_1(y_prime_a2, zeta_prime_a2, vertical_wing, inverted_wing)
+    y_prime_b1, zeta_prime_b1 = flip_1(y_prime_b1, zeta_prime_b1, vertical_wing, inverted_wing)
+    y_prime_bh, zeta_prime_bh = flip_1(y_prime_bh, zeta_prime_bh, vertical_wing, inverted_wing)
+    y_prime_bc, zeta_prime_bc = flip_1(y_prime_bc, zeta_prime_bc, vertical_wing, inverted_wing)
+    y_prime_b2, zeta_prime_b2 = flip_1(y_prime_b2, zeta_prime_b2, vertical_wing, inverted_wing)
+    y_prime_ch, zeta_prime_ch = flip_1(y_prime_ch, zeta_prime_ch, vertical_wing, inverted_wing)
+    y_prime   , zeta_prime    = flip_1(y_prime   , zeta_prime   , vertical_wing, inverted_wing)
+
+    # Deflect control surfaces-----------------------------------------------------------------------------
+    # note:    "positve" deflection corresponds to the RH rule where the axis of rotation is the OUTBOARD-pointing hinge vector
```

Comment:
is there documentation on this somewhere, it would be nice to have an infographic somewhere on the website.